src/soilbiogeochem/SoilBiogeochemDecompCascadeConType.F90

```diff
@@ -37,6 +42,27 @@ module SoilBiogeochemDecompCascadeConType
      logical           , pointer  :: is_cellulose(:)                   ! TRUE => pool is cellulose
      logical           , pointer  :: is_lignin(:)                      ! TRUE => pool is lignin
      real(r8)          , pointer  :: spinup_factor(:)                  ! factor by which to scale AD and relevant processes by
+
+     integer,pointer :: spm_tranlist_a(:,:)
```

Comment:
These variables need comments to define what they are

Reply:
done